src/app/features/charts/components/score-barchart/score-barchart.component.ts

```diff
@@ -45,26 +45,32 @@ export class ScoreBarChartComponent implements AfterViewInit, OnChanges, OnDestr
   constructor(private helperService: HelperService) {
   }
 
-  ngOnChanges(): void {
-    if (this.initialized) {
-      // if chart has already been initialized, this means the data has changed
-      this.createChart();
+  ngOnChanges(changes: SimpleChanges): void {
+    if (
```

Comment:
we need to unfortunately check firstChange === false because on initial load, we don't want to draw the chart twice.  Also, ngOnChanges fires on initial load and subsequent loads